packages/eslint-plugin/lib/config/recommended.js

```diff
@@ -185,6 +185,7 @@ module.exports = {
     '@department-of-veterans-affairs/prefer-telephone-component': 1,
     '@department-of-veterans-affairs/telephone-contact-digits': 1,
     '@department-of-veterans-affairs/deprecated-classes': 1,
+    '@department-of-veterans-affairs/use-new-utility-classes': 1,
```

Comment:
@jhonnyoddball Do we want this set to 1 already?

Reply:
Yes, as recommended by default. But I will turn it off once it's installed in the `vets-website`

Reply:
Sounds good. Can we make sure that there is a follow-up ticket (or task added to a ticket) that reminds us to turn it on in vets-website when we're ready? Carol might be able to help figure out where that should be added.

Reply:
After thinking about it, I'm going to remove it from the recommended so we don't have to turn it off. And when we are ready we can just upgrade to dependency and manually turn it on. 
This is just in case this package is installed in another repo.